core/quantity/src/test/TestUtils/TestHelper.ts

```diff
@@ -153,6 +158,12 @@ export class TestUnitsProvider implements UnitsProvider, AlternateUnitLabelsProv
       const conversionData = new ConversionData();
       conversionData.factor = deltaNumerator / deltaDenominator;
       conversionData.offset = deltaOffset;
+      if (fromUnitData.isInverted && !toUnitData.isInverted) {
```

Comment:
@rschili do we want to update core-frontend's basicUnitsProvider to reflect this too...?

Reply:
I thought about this, too. For now, BasicUnitsProvider has no inverted units, it also does not have the SLOPE Phenomenon. And we want to eventually deprecate it for the SchemaUnitsProvider. So I think this is fine.